src/test/run-pass-fulldeps/qquote.rs

```diff
@@ -9,59 +9,56 @@
 // except according to those terms.
 
 // ignore-cross-compile
-// ignore-pretty
 
 #![feature(quote, rustc_private)]
 
 extern crate syntax;
 
-use syntax::ast;
-use syntax::codemap;
-use syntax::parse;
-use syntax::print::pprust;
+use syntax::codemap::DUMMY_SP;
+use syntax::print::pprust::*;
 
-trait FakeExtCtxt {
```

Comment:
Um, I changed the main function for a reason - I'm testing unquotes now (which is what this branch is all about). I'll change the other files that use a FakeExtCtxt, though.